src/ProductConstructionService/ProductConstructionService.Cli/Operations/DeploymentOperation.cs

```diff
@@ -126,30 +126,38 @@ public async Task<int> RunAsync()
         }
     }
 
-    private async Task CleanupRevisionsAsync(IEnumerable<ContainerAppRevisionTrafficWeight> revisionsTrafficWeight)
+    private async Task RemoveRevisionLabel(string revisionName, string? label)
     {
-        // Cleanup all revision labels
-        foreach (var revisionTrafficWeight in revisionsTrafficWeight)
+        if (!string.IsNullOrEmpty(label))
```

Comment:
Why not just make the function accept `string label` if it does not do anything with `null`? Seems wrong. The compiler would then force you to put the if where you call this which is more natural